src/sentry/interfaces/threads.py

```diff
@@ -1,44 +1,47 @@
 from sentry.interfaces.base import Interface
 from sentry.interfaces.stacktrace import Stacktrace
 from sentry.utils.json import prune_empty_keys
-from sentry.utils.safe import trim
+from sentry.utils.safe import get_path, trim
 
 __all__ = ("Threads",)
 
 
-def get_stacktrace(value, raw=False):
```

Comment:
So the `raw` keyword arg isn't used anywhere, and won't be passed on to `Interface.to_python()` accidentally?